glassfish-on-suse/azuredeploy.parameters.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "_artifactsLocation": {
+      "value": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/glassfish-on-suse/"
+    },
+    "glassfishVersion": {
+      "value": 3
+    },
+    "glassfishRelease": {
+      "value": "3.1.2.2"
+    },
+    "numberOfInstances": {
+      "value": 2
+    },
+    "dnsNameforLBIP": {
+      "value": "GEN-UNIQUE"
+    },
+    "vmSize": {
+      "value": "Standard_DS2_v2"
+    },
+    "osImage": {
+      "value": "opensuse"
+    },
+    "adminUsername": {
```

Comment:
I have done this, but the comment has not been removed as it was placed on the line above the value